### ISSUE TITLE
New MR Launcher for windows has 0 screen reader compatibility

### ISSUE THREAD

**Issue by BrailleBennett** [modrinth/code#2282]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [X] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

I can't do this

### What operating systems are you seeing the problem on?

Windows

### Describe the bug

If you try to navigate around the modrinth launcher on windows with a screen reader like [NVDA](https://nvaccess.org) or Windows Narrator it looks like they keys are being passed through to the  app and the screen readerdoesn't read anything. The screen reader acts as if there are 0 buttons on the screen when in reality the main page of the launcher is open and there are hundreds of clickable elements.

### Steps to reproduce

1. Enable a screen reader like NVDA or Windows Narrator
2. Open the modrinth launcher app
3. Try tabbing around or clicking the space bar/enter key to activate an element.
4. Observe how the screen reader is completely silent and doesn't interact with the app at all.

### Expected behavior

The screen reader reads each button independently and reports what the curser is hovering over as well as when the screen changes.

**Comment by brawaru** [comment]
Using NVDA it does indeed not read anything, but when I click on the window it starts working. However, there a lot of accessibility issues in the app, e.g. instance view cannot be tabbed at all, and many buttons and inputs are unlabelled.

**Comment by BrailleBennett** [comment]
I was able to click on the window and it did start reading, this is a weird extra step though to say the least, and no other app suffers from this. I also noticed that images don't get labels until at least one thing has been clicked, E.G.: the library is called graphic by NVDA until you press a button that refreshes screen.

**Comment by Felix14-v2** [comment]
Windows Narrator works just fine for me (Windows 11), at least when I use arrows or tab to navigate.
